src/app/dataset-view/additional-components/overview-component/overview-component.html

```diff
@@ -6,32 +6,154 @@
             >
-                <ng-container
-                    *ngIf="currentState.overview.metadata.currentInfo?.description"
+                <div class="d-flex flex-column me-auto">
+                    <div
+                        *ngIf="currentState.overview.metadata.currentInfo?.description"
+                    >
+                        <h2 class="box-title align-items-center pb-3">
+                            Description
+                        </h2>
+
+                        <p>
+                            {{
+                            currentState.overview.metadata.currentInfo.description
+                            }}
+                        </p>
+                    </div>
+                    <div
+                        class="app-mat-chip"
+                        *ngIf="currentState.overview.metadata.currentInfo?.keywords"
+                    >
+                        <h2 class="box-title align-items-center pt-3">
+                            Keywords
+                        </h2>
+                        <mat-chip-list aria-label="Tags selection">
+                            <div
+                                *ngFor="let keyword of currentState.overview.metadata.currentInfo?.keywords"
+                            >
+                                <mat-chip
+                                    (click)="selectTopic(datasetBasics.kind)"
+                                    >{{ keyword }}</mat-chip
+                                >
+                            </div>
+                        </mat-chip-list>
+                    </div>
+                </div>
+                <div
+                    *ngIf="currentState.overview.metadata.currentInfo.description || currentState.overview.metadata.currentInfo.keywords?.length"
                 >
-                    <h2 class="box-title align-items-center pb-3">
-                        Description
-                    </h2>
-                    <p>
-                        {{
-                        currentState.overview.metadata.currentInfo.description
-                        }}
-                    </p>
-                </ng-container>
-                <div class="app-mat-chip">
-                    <mat-chip-list aria-label="Tags selection">
-                        <div
-                            *ngFor="let keyword of currentState.overview.metadata.currentInfo?.keywords"
+                    <button
+                        class="btn-sm btn custom-edit"
+                        aria-label="Edit dataset details"
+                        data-test-id="edit-dataset-details"
+                        title="Edit dataset details"
+                        (click)="openInformationModal(content)"
+                    >
+                        <svg-icon name="pencil"></svg-icon>
+                    </button>
+                </div>
+            </div>
+            <ng-template #content let-modal>
```

Comment:
Please move the modal window to a separate component

Reply:
Moved  the modal window to a separate component